provision/pull-images.sh

```diff
@@ -21,23 +22,31 @@ if [ -z "${NAME_PREFIX}" ]; then
         docker.io/cilium/connectivity-container:v1.0 \
         docker.io/cilium/demo-client:latest \
         docker.io/cilium/demo-httpd:latest \
+        docker.io/cilium/istio_pilot:1.0.2 \
         docker.io/cilium/kafkaclient2:latest \
         docker.io/cilium/kafkaclient:latest \
         docker.io/cilium/microscope:1.1.2-ci \
         docker.io/cilium/starwars:v1.0 \
         docker.io/coredns/coredns:1.2.2 \
         docker.io/coredns/coredns:1.0.6 \
         docker.io/digitalwonderland/zookeeper:latest \
+        docker.io/istio/citadel:1.0.2 \
+        docker.io/istio/galley:1.0.2 \
+        docker.io/istio/proxyv2:1.0.2 \
+        docker.io/istio/mixer:1.0.2 \
         docker.io/istio/examples-bookinfo-details-v1:1.6.0 \
         docker.io/istio/examples-bookinfo-productpage-v1:0.2.3 \
         docker.io/istio/examples-bookinfo-ratings-v1:1.6.0 \
         docker.io/istio/examples-bookinfo-reviews-v1:1.6.0 \
         docker.io/istio/examples-bookinfo-reviews-v2:1.6.0 \
+        docker.io/istio/proxy_init:1.0.2 \
         docker.io/kubernetes/guestbook:v2 \
         docker.io/library/busybox:1.28.0 \
         docker.io/library/httpd:2.4.34 \
         docker.io/library/redis:4.0.11 \
         docker.io/library/registry:2.6.2 \
+        docker.io/prom/prometheus:v2.3.1 |
```

Comment:
There is a typo here, you used `|` instead `\`

Reply:
*facepalm* :)